### PR TITLE
Split testing the installers into three steps

### PR DESCRIPTION
**Pull request by PaulWessel**
Also remove News from the edits needed in the website repo.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release_checklist.md [hunk 1]
```diff
@@ -41,7 +41,9 @@ assignees: ''
 - [ ] create source tarballs (tar.gz and tar.xz) (@PaulWessel)
 - [ ] create macOS bundle (@PaulWessel)
 - [ ] create Windows installers (win32 and win64) (@joa-quim)
-- [ ] check if the source tarballs, macOS bundle and Windows installers work well
+- [ ] check if the source tarballs for Linux work well (@Esteban82, @anbj)
+- [ ] check if the macOS bundles work well (@seisman, @meghanrjones)
+- [ ] check if the Windows installers work well (@seisman)
```
Comment (seisman) [inline .github/ISSUE_TEMPLATE/release_checklist.md:46]: I don't have a Windows machine, so can't test the Windows installers.

Comment (PaulWessel) [reply .github/ISSUE_TEMPLATE/release_checklist.md:46]: OK, wasn't sure.  Do we know anyone we could rope into helping with such a check?

Comment (PaulWessel) [reply .github/ISSUE_TEMPLATE/release_checklist.md]: Changed it to (volunteers needed!)

**Comment by PaulWessel** [comment]
@Esteban82, could you please edit the instagram entry on this checklist?

**Comment by Esteban82** [comment]
> @Esteban82, could you please edit the instagram entry on this checklist?

Done!

**Comment by PaulWessel** [comment]
Thanks, awaiting reviews by the others.

**Review by meghanrjones — APPROVED** [review]

**Pull request merged**